src/test/java/org/folio/marc4ld/mapper/MarcBib2LdMapperIT.java

```diff
@@ -214,17 +215,18 @@ void map_shouldReturnCorrectlyMappedEmptyResource() {
     var marc = loadResourceAsString("empty_marc.jsonl");
 
     // when
-    var result = marc2BibframeMapper.fromMarcJson(marc);
+    var result = marcBibToResource(marc);
 
     // then
     assertThat(result).isNotNull();
-    validateAllIds(result);
     assertThat(result.getLabel()).isEmpty();
     assertThat(result.getDoc()).isEmpty();
-    assertThat(result.getInventoryId()).isNull();
-    assertThat(result.getSrsId()).isNull();
     assertThat(result.getTypes()).containsOnly(INSTANCE);
     assertThat(result.getOutgoingEdges()).isEmpty();
+    var instanceMetadata = result.getInstanceMetadata();
+    assertThat(instanceMetadata.getInventoryId()).isNull();
+    assertThat(instanceMetadata.getSrsId()).isNull();
+    assertThat(instanceMetadata.getSource()).isEqualTo(MARC);
```

Comment:
looks like this check is already included in ```marcBibToResource``` method (same in some other test cases below)

Reply:
Done